### PR TITLE
Applied feedback to the moving-levels review template

### PR DESCRIPTION
**Pull request by eddie-knight**

### CODE REVIEW

**Comment by eddie-knight** [comment]
cc/ @brandtkeller

File: project-resources/moving-levels-review-template.md [hunk 1]
```diff
@@ -8,13 +8,17 @@ What ecosystem adoption has the project seen?
 
 ### Past TOC Reviews
 
-How has the project addressed comments from previous reviews (incubation if graduation, sandbox if incubating, etc)?
+If already accepted, how has the project addressed comments from previous TAG or TOC reviews?
```
Comment (mnm678) [inline project-resources/moving-levels-review-template.md:11]:
```suggestion
If the project has undergone a previous TAG or TOC review, how has the project addressed comments from those reviews?
```

**Review by mnm678 — COMMENTED** [review]
One nit, but overall this is a good clarification

**Comment by brandtkeller** [comment]
Link check is failing and should be fixed in #1403 - we can merge that PR and use this one after updating to ensure the changes worked as expected.

**Pull request merged**